packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/Sell/index.js

```diff
@@ -24,7 +24,7 @@ class SellContainer extends React.Component {
     this.props.coinifyActions.saveMedium('blockchain')
   }
 
-  startSell () {
+  startSell (values) {
```

Comment:
Looks like `values` is passed in but not used. Could be confusing for someone else looking at this.